src/main/java/com/fnp/Webapp Fundamentals/newcrud/src/main/java/com/fnp/controller/MainController.java

```diff
@@ -0,0 +1,33 @@
+package com.fnp.controller;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import com.fnp.service.InsertService;
+
+/**
+ * 
+ * Class for insert function.
+ * @author Saransh vijay
+ *
+ */
+@WebServlet("/Main")
+public class MainController extends HttpServlet {
+
+	private static final long serialVersionUID = 1L;
+
+	@Override
+	protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+
+		/**
```

Comment:
if you want you can add single line comment for what we are doing here briefly..but no documentation needed here.